config/charts/inferencepool/values.yaml

```diff
@@ -40,6 +40,17 @@ inferenceExtension:
 
   tolerations: []
 
+  # Monitoring configuration for EPP
+  monitoring:
+    interval: "10s"
```

Comment:
nit: Make the default scraping interval `15s` instead of `10s`.

See https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work that the `rate` query should have a range four times the scrape interval. The most common range in the query is `1m`, hence setting interval to `15s` will be better.